### PR TITLE
Fix/15673692 description not required

### PR DESCRIPTION
**Pull request by CzarekDryl**
https://tw.auroracreation.com/app/projects/562144/tasks/15685797

Changes in this PR:
- Description field is no longer required

### CODE REVIEW

**Review by melyndav — CHANGES_REQUESTED** [review]
Hey @CzarekDryl looks great from my end, so thanks for your work on this.

Two requests:

1. Can you please leave the description as required only for the 'Create a decision' action type? The description field is the decision content. 

2. For this 'Create a decision' error states validation copy can the following errors please be updated:

- decisionMethod must be defined > Please select a decision method 
- description must be defined > Please enter a description for your agreement 

Thank you!

**Comment by CzarekDryl** [comment]
@melyndav Sure thing! I've changed that

File: src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/hooks.ts [hunk 1]
```diff
@@ -15,14 +15,14 @@ import {
...
 } from '~v5/common/ActionSidebar/consts';
-import { MAX_ANNOTATION_NUM } from '~v5/shared/RichText/consts';
 
 import { ActionFormBaseProps } from '../../../types';
 import {
   DecisionMethod,
   DECISION_METHOD,
   useActionFormBaseHook,
 } from '../../../hooks';
+import { MAX_ANNOTATION_LENGTH } from '~constants';
```
Comment (thomassdk) [inline src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/hooks.ts:25]: this needs to go up :point_up:

**Review by thomassdk — APPROVED** [review]
Looks good to me, just minor issue with imports

**Review by ArmandoGraterol — APPROVED** [review]
Looks good to me! Descriptions are indeed, no longer required 👍

**Pull request merged**